CMakeLists.txt

```diff
@@ -16,6 +16,15 @@ endif (APPLE)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/)
 
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
+    set (OSG_PLUGINS_DIR CACHE STRING "")
+    find_package(Freetype REQUIRED)
```

Comment:
FindMyGUI.cmake already handles finding of freetype when MYGUI_STATIC is set. Or at least it should.